opcua/server/internal_server.py

```diff
@@ -86,20 +86,29 @@ def setup_nodes(self):
         ns_node = Node(self.isession, ua.NodeId(ua.ObjectIds.Server_NamespaceArray))
         ns_node.set_value(uries)
 
-    def load_standard_address_space(self, cacheFile=None):
-        if cacheFile and path.isfile(cacheFile):
+    def load_standard_address_space(self, cachefile=None):
+        # check for a cache file, in windows the file extension is also needed for the check
+        cachefile_win = cachefile
+        if cachefile_win:
+            cachefile_win += ".dat"
+
+        if cachefile and (path.isfile(cachefile) or path.isfile(cachefile_win)):
```

Comment:
if XXX and (YYY or ZZZ)

with () otherwise you will not get expected result...